### ISSUE TITLE
improve mousedrag movement in FreeFlight Controls when mouse-dragging near borders of scene's canvas

### ISSUE THREAD

**Issue by cvan** [mozilla/unity-webvr-export#219]
originally identified in PR #213 (for issue #125) in this comment: https://github.com/mozilla/unity-webvr-export/pull/213#discussion_r177841935:

> not sure if it's coming from the `YawMovement` or `PitchMovement`, but the mousedrag (especially at lower regions of the screen) doesn't seem to rotate the camera correctly; it's like a lag or stutter

**Comment by caseyyee** [comment]
Yeah, sem to get stuttering and some inconsistencies when dragging.

Maybe something more along the lines of: 
https://threejs.org/examples/misc_controls_pointerlock.html

It's a bit different in that it's not freeflight, but the mouse-locked, look and fps style controls seems a bit more what I would expect in terms of navigating a standard scene.

**Comment by cvan** [comment]
I also notice that the controls keep moving even after I have released a mouse click. I would agree that I would expect FPS Pointer-Lock as a default, preferred camera controls. Free-Flight is useful, but I'm not sure the demo scene is a good example of that. (out of curiosity, what are some examples of scenes where Free-Flight would be preferred?)
